paper/main.tex

```diff
@@ -520,7 +524,7 @@ \subsubsection{Configuring and querying Davos}\label{subsec:config}
 
 \end{itemize}
 
-\noindent The current values of all \texttt{davos} attributes may be viewed at once within a notebook by displaying the \texttt{davos.config} object.
+\noindent The current values of all \texttt{davos} attributes may be viewed at once within a notebook by printing the \texttt{davos.config} object.
```

Comment:
"displaying" $\neq$ "printing" in IPython/a Jupyter notebook. 
I was trying to convey that you can just type `davos.config` in a notebook cell, run it, and view the current config values. When you do that in IPython or a Jupyter notebook, the output you're shown is not equivalent to `print(foo)`, it's equivalent to
```python
from IPython.display import display
display(foo)
```
A `pandas.DataFrame` is a good example of this.

I'd like to keep this distinction in there (i.e., revert "printing" to "displaying") because even though the output of displaying and printing `davos.config` is currently the same, it likely won't be in the future. The difference between printing & displaying in a notebook comes from the fact that printing an object shows the output of its `__str__()` method (which, if not defined, falls back to its `__repr__()` method), while displaying shows the output of the object's `_repr_html()` method if one is defined, and falls back to its `__repr__()` if not. Since the purpose of `davos.config` is now pretty much entirely for inspecting config options rather than setting them, I'm planning to add a `_repr_html()` method to make this look nicer

Reply:
got it.  that's fine then